redbot/cogs/streams/streams.py

```diff
@@ -619,24 +640,27 @@ async def ignore_reruns(self, ctx: commands.Context):
             await self.config.guild(guild).ignore_reruns.set(True)
             await ctx.send(_("Streams of type 'rerun' will no longer send an alert."))
 
-    async def add_or_remove(self, ctx: commands.Context, stream):
-        if ctx.channel.id not in stream.channels:
-            stream.channels.append(ctx.channel.id)
+    async def add_or_remove(self, ctx: commands.Context, stream, discord_channel):
+        if discord_channel is None:
+            discord_channel = ctx.channel
+
+        if discord_channel.id not in stream.channels:
+            stream.channels.append(discord_channel.id)
             if stream not in self.streams:
                 self.streams.append(stream)
             await ctx.send(
                 _(
-                    "I'll now send a notification in this channel when {stream.name} is live."
-                ).format(stream=stream)
+                    "I'll now send a notification in the channel {channel} when {stream.name} is live."
+                ).format(stream=stream, channel=discord_channel)
             )
         else:
-            stream.channels.remove(ctx.channel.id)
+            stream.channels.remove(discord_channel.id)
             if not stream.channels:
                 self.streams.remove(stream)
             await ctx.send(
                 _(
-                    "I won't send notifications about {stream.name} in this channel anymore."
-                ).format(stream=stream)
+                    "I won't send notifications about {stream.name} in the channel {discord_channel} anymore"
```

Comment:
Channel mention would look better here. I also added a dot at the end and re-ordered the channel mention and "channel" word in the code suggestion.
```suggestion
	                    "I won't send notifications about {stream.name}"
                        " in the {channel.mention} channel anymore."
```